src/docs/output-targets/custom-elements.md

```diff
@@ -115,6 +113,8 @@ Also be sure to set `@stencil/core` as a dependency of the package.
 
 Note: If you are distributing both the `dist` and `dist-custom-elements`, then it's up to you to choose which one of them should be available in the `module` entry.
```

Comment:
unrelated to your change: I've noticed that we often put notes like this inside of a blockquote (`> Note: ...`) so they stand out a bit.

Would it make sense to do that while we're in here? If we wanted to, we might want to move line 116 above the note

Reply:
👍 Done in https://github.com/ionic-team/stencil-site/pull/886/commits/67bc5a8636270d91f012be5f0d7b949a0b6c7c03 

![Screen Shot 2022-05-24 at 1 14 37 PM](https://user-images.githubusercontent.com/1930213/170094017-1aca8935-a34c-4bdf-9a29-186a71cfbc02.png)

Reply:
beautiful!